mmcv/transforms/loading.py

```diff
@@ -35,23 +36,50 @@ class LoadImageFromFile(BaseTransform):
             Defaults to 'cv2'.
         file_client_args (dict): Arguments to instantiate a FileClient.
             See :class:`mmengine.fileio.FileClient` for details.
-            Defaults to ``dict(backend='disk')``.
+            Defaults to None. It will be deprecated in future. Please use
+            ``backend_args`` instead.
+            Deprecated in version 2.0.0rc4.
         ignore_empty (bool): Whether to allow loading empty image or file path
             not existent. Defaults to False.
+        backend_args (dict, optional): Arguments to instantiate the
+            preifx of uri corresponding backend. Defaults to None.
+            New in version 2.0.0rc4.
     """
 
     def __init__(self,
                  to_float32: bool = False,
                  color_type: str = 'color',
                  imdecode_backend: str = 'cv2',
-                 file_client_args: dict = dict(backend='disk'),
-                 ignore_empty: bool = False) -> None:
+                 file_client_args: Optional[dict] = None,
+                 ignore_empty: bool = False,
+                 backend_args: Optional[dict] = None) -> None:
         self.ignore_empty = ignore_empty
         self.to_float32 = to_float32
         self.color_type = color_type
         self.imdecode_backend = imdecode_backend
+
+        if file_client_args is not None:
+            warnings.warn(
+                '"file_client_args" will be deprecated in future. '
+                'Please use "backend_args" instead', DeprecationWarning)
+            if backend_args is not None:
+                raise ValueError(
+                    '"file_client_args" and "backend_args" cannot be set '
+                    'at the same time.')
+        else:
+            file_client_args = dict(backend='disk')
+        if backend_args is None:
+            backend_args = dict(backend='disk')
```

Comment:
Can we set these arguments to None by default?

Reply:
OK